README.md

```diff
@@ -82,6 +84,16 @@ The master branch works with **PyTorch 1.5+**.
 
 **Release**: v0.24.0 was released in 05/05/2022. Please refer to [changelog.md](docs/changelog.md) for details and release history.
 
+### 🌟 Preview of 1.x version
```

Comment:
It's better to use 1.0 instead of 1.x here.

And what about moving 1.0 information ahead of the stable version?